mkdocs_static_i18n/utils.py

```diff
@@ -26,3 +27,17 @@ def get_plugin(name: str, config: MkDocsConfig) -> Optional[Plugin]:
             return p
 
     return None
+
+
+class i18nLoggingFilter:
```

Comment:
```suggestion
class I18nLoggingFilter:
```

I'd rather classes start with a uppercase letter please

Reply:
Me too, I just forgot myself out of habit of writing `i18n` in that case. Good catch. I think Pylint detects those kinds of name issues, maybe it's worth adding it to the hatch style run 🤷